packages/rocketchat-ui-message/message/popup/messagePopup.coffee

```diff
@@ -160,7 +160,7 @@ Template.messagePopup.onCreated ->
 
 	template.records = new ReactiveVar []
 	Tracker.autorun ->
-		if template.data.collection.find?
+		if template.data.collection.findOne?
 			template.data.collection.find().count()
```

Comment:
Also, it looks like it is not doing anything, but I may be wrong for not seeing the whole picture